docs.md

```
@@ -0,0 +1,184 @@
+- [ ] https://redux.js.org/introduction/getting-started
+- [ ] https://reactrouter.com/home
+- [ ] https://docs.pi-hole.net/
+- [ ] https://guacamole.apache.org/doc/gug/
+- [ ] https://kotlinlang.org/docs/home.html
+- [ ] https://openzfs.readthedocs.io/en/latest/introduction.html
+- [ ] https://www.zfsnap.org/docs.html
+- [ ] https://docs.firecrawl.dev/introduction
+- [ ] https://filebrowser.org/installation
+- [ ] https://www.filestash.app/docs/?origin=menu
+- [ ] https://kometa.wiki/en/latest/
+- [ ] https://freshrss.github.io/FreshRSS/en/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)